### DISCUSSION TITLE
Firestore & Realtime DB empty

### DISCUSSION THREAD

**Discussion by ferhat37ozkan** [TheAlphamerc/flutter_twitter_clone#201] [Q&A]
Hello, Account creating working but my firestore and realtime DB looks empty. How  can I make it work? 

W/SyncTree: Listen at /tweet failed: DatabaseError: Permission denied
W/SyncTree: Listen at /profile failed: DatabaseError: Permission denied
W/SyncTree: Listen at /notification/oLUIUoIya5NFAwZwLlsepdD650A2 failed: DatabaseError: Permission denied
W/SyncTree: Listen at /chatUsers/oLUIUoIya5NFAwZwLlsepdD650A2 failed: DatabaseError: Permission denied
    Listen at /chats/oLUIU-oLUIU failed: DatabaseError: Permission denied
W/SyncTree: Listen at /profile/oLUIUoIya5NFAwZwLlsepdD650A2 failed: DatabaseError: Permission denied
E/flutter: [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: DatabaseError(-3, Permission denied, )
I/chatty: uid=10160(com.batanay.adoptme) 1.ui identical 11 lines
E/flutter: [ERROR:flutter/lib/ui/ui_dart_state.cc(186)] Unhandled Exception: DatabaseError(-3, Permission denied, )
W/System: Ignoring header X-Firebase-Locale because its value was null.
D/FirebaseAuth: Notifying id token listeners about user ( oLUIUoIya5NFAwZwLlsepdD650A2 ).

Thank with my regards

**Comment by ferhat37ozkan** [comment]
![ScreenShot](https://user-images.githubusercontent.com/41229777/126834206-188e9be9-07e5-4124-8a5c-bd92f4079a1a.PNG)

**Reply by TheAlphamerc** [reply]
It is because of a permission issues. Once you set the realtime | firestore security rules then you will be able to read/write data in database.
![Screenshot 2021-07-24 at 9 44 21 AM](https://user-images.githubusercontent.com/37103237/126857027-e225bccb-d710-40ed-8abb-ec1c2c79be2f.png)

**Comment by programmingjoomla2011** [comment]
I cannot find JSON files for 'realtime' and 'firestore'. Will you please share it to here?

**Reply by TheAlphamerc** [reply]
do you mean googleservice.json file ?
